runtime/cere-dev/src/lib.rs

```diff
@@ -1401,7 +1403,13 @@ pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
 /// Runtime migrations
-type Migrations = ();
+type Migrations = (
+	pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
+	pallet_ddc_clusters::migrations::v2::MigrateToV2<Runtime>,
+	pallet_ddc_staking::migrations::v1::MigrateToV1<Runtime>,
+	pallet_ddc_customers::migration::MigrateToV2<Runtime>,
+	pallet_ddc_customers::migration::MigrateToV1<Runtime>,
```

Comment:
`pallet_ddc_customers::migration::MigrateToV1` has been executed already in `cere_dev`. You should remove it from the list.